app/services/proofing/lexis_nexis/request.rb

```diff
@@ -5,18 +5,19 @@ class RequestError < StandardError; end
     class Request
       attr_reader :config, :applicant, :url, :headers, :body
 
-      def initialize(config:, applicant:)
+      def initialize(config:, applicant:, require_auth_headers: true)
         @config = config
         @applicant = applicant
+        @require_auth_headers = require_auth_headers
         @body = build_request_body
         @headers = build_request_headers
         @url = build_request_url
       end
 
-      def send
+      def send_request
```

Comment:
To be able to call the private method Proofing::LexisNexis::Ddp::VerificationRequest#build_request_headers, I needed to rename Proofing::LexisNexis::Request#send to stop overriding Object#send